bin/runtime/src/lib.rs

```diff
@@ -905,37 +899,6 @@ impl pallet_feature_control::Config for Runtime {
 }
 
 // Create the runtime by composing the FRAME pallets that were previously configured.
-#[cfg(not(feature = "liminal"))]
-construct_runtime!(
-    pub struct Runtime {
-        System: frame_system = 0,
-        RandomnessCollectiveFlip: pallet_insecure_randomness_collective_flip = 1,
-        Scheduler: pallet_scheduler = 2,
-        Aura: pallet_aura = 3,
-        Timestamp: pallet_timestamp = 4,
-        Balances: pallet_balances = 5,
-        TransactionPayment: pallet_transaction_payment = 6,
-        Authorship: pallet_authorship = 7,
-        Staking: pallet_staking = 8,
-        History: pallet_session::historical = 9,
-        Session: pallet_session = 10,
-        Aleph: pallet_aleph = 11,
-        Elections: pallet_elections = 12,
-        Treasury: pallet_treasury = 13,
-        Vesting: pallet_vesting = 14,
-        Utility: pallet_utility = 15,
-        Multisig: pallet_multisig = 16,
-        Sudo: pallet_sudo = 17,
-        Contracts: pallet_contracts = 18,
-        NominationPools: pallet_nomination_pools = 19,
-        Identity: pallet_identity = 20,
-        CommitteeManagement: pallet_committee_management = 21,
-        Proxy: pallet_proxy = 22,
-        FeatureControl: pallet_feature_control = 23,
-    }
-);
-
-#[cfg(feature = "liminal")]
 construct_runtime!(
     pub struct Runtime {
```

Comment:
```
VkStorage: pallet_vk_storage = 41,
```
do we leave it as it is? we still have option to give it `24`, ie next consecutive number, since this change is not released anywhere yet. `VkStorage` is now standard pallet of our runtime.

Reply:
decreased to 41, aleph-client regenerated